docs/reference/settings/inference-settings.asciidoc

```diff
@@ -0,0 +1,92 @@
+
+`xpack.inference.http.max_total_connections`::
+(<<cluster-update-settings,Dynamic>>) Specifies the maximum number of connections the internal connection pool can
+lease. Defaults to `50`.
+
+`xpack.inference.http.max_route_connections`::
+(<<cluster-update-settings,Dynamic>>) Specifies the maximum number of connections a single route can lease from
+the internal connection pool. If this setting is set to a value equal to or greater than
+`xpack.inference.http.max_total_connections`, then a single third party service could lease all available
+connections and other third party services would be unable to lease connections. Defaults to `20`.
+
+`xpack.inference.http.connection_eviction_interval`::
+(<<cluster-update-settings,Dynamic>>) Specifies the interval that an eviction thread will run to remove expired and
+stale connections from the internal connection pool. Decreasing this time value can help improve throughput if
+multiple third party service are contending for the available connections in the pool. Defaults to one minute (`1m`).
+
+`xpack.inference.http.connection_eviction_max_idle_time`::
+(<<cluster-update-settings,Dynamic>>) Specifies the maximum duration a connection can be unused before it is marked as
+idle and can be closed and removed from the shared connection pool. Defaults to one minute (`1m`).
+
+`xpack.inference.http.request_executor.queue_capacity`::
+(<<cluster-update-settings,Dynamic>>) Specifies the size of the internal queue for requests waiting to be sent. If
+the queue is full and a request is sent to the inference API, it will be rejected. Defaults to `2000`.
+
+[[xpack-inference-http-retry-settings]]
+==== Inference API HTTP Retry settings
+
+When a third party service returns a failure code that is transient (e.g. 429), the request is retried by the inference
+API. These settings govern the retry behavior. When a request is retried, exponential backoff is used.
+
+`xpack.inference.http.retry.initial_delay`::
+(<<cluster-update-settings,Dynamic>>) Specifies the initial delay before retrying a request. Defaults to one second
+(`1s`).
+
+`xpack.inference.http.retry.max_delay_bound`::
+(<<cluster-update-settings,Dynamic>>) Specifies the maximum delay for a request. Defaults to five seconds (`5s`).
+
+`xpack.inference.http.retry.timeout`::
+(<<cluster-update-settings,Dynamic>>) Specifies the maximum amount of time a request can be retried.
+Once the request exceeds this time, the request will no longer be retried and a failure will be returned.
+Defaults to 30 seconds (`30s`).
+// end::inference-logging[]
+
+[[xpack-inference-input-text]]
+// tag::inference-input-text[]
+==== Inference API Input text
+
+For certain 3rd party service integrations, when the service returns an error indicating that the request
+input was too large, the input will be truncated and the request is retried. These settings govern
+how the truncation is performed.
+
+`xpack.inference.truncator.reduction_percentage`::
```

Comment:
This isn't actually the name of this setting. There was a typo that is being corrected in this PR: https://github.com/elastic/elasticsearch/pull/108219/files#diff-24707283cf310e9531dd805fdbaf557c7847f2cc63642ccc8d3fe8d5e80741d3R29